SD_Assignment1/src/main/java/ro/utcn/aron/model/Question.java

```diff
@@ -1,21 +1,55 @@
 package ro.utcn.aron.model;
 
-public class Question {
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class Question implements Comparable {
 
 	private int id;
+	private int score = 0;
 	private String title;
 	private String body;
 	private String tags;

```

Comment:
It would have been better to have either a list of Strings or a list of Tag entities instead of a single string (which contains the tags concatenated).